maistra/bazelrc-vendor

```diff
@@ -51,6 +51,7 @@ build --override_repository=com_googlesource_chromium_v8=/work/maistra/vendor/co
 build --override_repository=com_googlesource_code_re2=/work/maistra/vendor/com_googlesource_code_re2
 build --override_repository=com_lightstep_tracer_cpp=/work/maistra/vendor/com_lightstep_tracer_cpp
 build --override_repository=envoy=/work/maistra/vendor/envoy
+build --override_repository=envoy_api=/work/maistra/vendor/envoy_api
```

Comment:
Is this right?

Reply:
You should probably rebuild the image (so that it gets the newer update-deps.sh) and run the command again.

Reply:
Thanks for catching this. Updated vendored dependencies.